src/Components/Address/__tests__/NewAddressForm.jest.tsx

```diff
@@ -0,0 +1,169 @@
+import { act, fireEvent, render, screen, waitFor } from "@testing-library/react"
+import { Button } from "@artsy/palette"
+
+import userEvent from "@testing-library/user-event"
+import {
+  AddressFormValues,
+  AddressFormWrapper,
+  AddressInputs,
+  EMPTY_ADDRESS,
+} from "Components/Address/NewAddressForm"
+import { Form } from "formik"
+
+const mockOnSubmit = jest.fn()
+
+const BasicAddressForm = props => {
+  return (
+    <AddressFormWrapper<AddressFormValues>
+      initialValues={EMPTY_ADDRESS}
+      onSubmit={mockOnSubmit}
+      {...props}
+    >
+      <Form>
+        <AddressInputs />
+        <Button type="submit">Submit</Button>
+      </Form>
+    </AddressFormWrapper>
```

Comment:
Using this as an example, its hard to see why this is better than just doing 
```jsx
<Formik ...>
  <Form>
    <AddressInputs>
  </Form>
</Formik>
```

Just because, all that `AddressFormWrapper` does is contain some default props, but it doesn't contain `<Form>`, it doesn't contain the form inputs, it doesn't contain submit buttons or handlers or anything like that. So all it appears to be is an intermediary-layer abstraction over a single other component (`<Formik />`) that basically does the exact same thing. 

If `AddressFormWrapper` contained more _behavior_ then I could understand its utility, but without that I would rather just have a file called `AddressFormInputs` and `addressFormValidation` and use those where need be. _OR_, if indeed the address form is a consistent, stable single thing, merge all of those together into one component as I mentioned elsewhere in this PR.